app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,14 @@
 <resources>
     <string name="app_name">moviO</string>
     <string name="main_activity_title">moviO</string>
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="overview_text">Two imprisoned men bond over a number of years, finding solace and eventual redemption through acts of common decency.</string>
+    <string name="movie_name_text">Movie name</string>
+    <string name="year_text">Year</string>
```

Comment:
why is there a text for "year" and "movie year" ? And this is for all strings below as well?